src/nvim/buffer_defs.h

```diff
@@ -151,9 +151,9 @@ typedef struct {
   char_u *wo_briopt;
 # define w_p_briopt w_onebuf_opt.wo_briopt /* 'breakindentopt' */
   int wo_diff;
-# define w_p_diff w_onebuf_opt.wo_diff  /* 'diff' */
-  long wo_fdc;
-# define w_p_fdc w_onebuf_opt.wo_fdc    /* 'foldcolumn' */
+# define w_p_diff w_onebuf_opt.wo_diff  ///< 'diff'
+  NumOpt wo_fdc;
+# define w_p_fdc w_onebuf_opt.wo_fdc    ///< 'foldcolumn'
```

Comment:
It appears that I have missed something about doxygen: `///<` is not for members, it is for “placing documentation after documented entity”. In any case `//` is not used inside macros.